pkg/apis/navigator/validation/cassandra.go

```diff
@@ -7,13 +7,24 @@ import (
 	"github.com/jetstack/navigator/pkg/apis/navigator"
 )
 
-func ValidateCassandraCluster(esc *navigator.CassandraCluster) field.ErrorList {
-	allErrs := ValidateObjectMeta(&esc.ObjectMeta, true, apimachineryvalidation.NameIsDNSSubdomain, field.NewPath("metadata"))
-	allErrs = append(allErrs, ValidateCassandraClusterSpec(&esc.Spec, field.NewPath("spec"))...)
+func ValidateCassandraCluster(c *navigator.CassandraCluster) field.ErrorList {
+	allErrs := ValidateObjectMeta(
+		&c.ObjectMeta,
+		true,
+		apimachineryvalidation.NameIsDNSSubdomain,
+		field.NewPath("metadata"),
+	)
+	allErrs = append(
+		allErrs,
+		ValidateCassandraClusterSpec(&c.Spec, field.NewPath("spec"))...,
+	)
```

Comment:
Can we inline these back into one line each? These functions are called fairly often, and the arguments aren't particularly complex (they are either naming simple parameters, or making a fairly small call to some other function)